protocols/jabber/iq.c

```diff
@@ -1040,6 +1051,8 @@ static xt_status jabber_iq_disco_server_response(struct im_connection *ic,
 		}
 	}
 
+	jabber_block_feature(ic);
```

Comment:
Hmm this function could use a better name, it looks weird. Does it.. block some feature? Does it feature blocking? We just don't know